map_tests/src/lib.rs

```diff
@@ -0,0 +1,36 @@
+// This crate contains tests for the map importing pipeline. They're closer to integration tests
+// than unit tests, mostly because of how tedious and brittle it would be to specify the full input
+// to individual pieces of the pipeline.
+
+#[cfg(test)]
+mod turns;
+
+// Run the contents of a .osm through the full map importer with default options.
+#[cfg(test)]
```

Comment:
I first tried specifying a `RawMap`, but the level of abstraction would kind of skip some of the OSM->RawMap stuff that we probably want to test, like filling out turn restrictions from relations.